entries/f/firsttechfed.com.json

```
@@ -9,6 +9,9 @@
       "call",
       "email"
     ],
+    "custom-software": [
+      "Entrust Identity Guard"
+    ],
```

After reviewing the data you've provided and what was previously provided at #5516 I think `Entrust Hardware Token` can be added as custom-hardware.
The previous submitter also mentioned sms, phone calls and email 2FA. Is that still something that is provided?

```suggestion
    ],
    "custom-hardware": [
      "Entrust Hardware Token"
    ],
```

---

Yes, the other data is still correct.